codespell_lib/data/dictionary.txt

```diff
@@ -25875,6 +25877,7 @@ substituded->substituted
 substitudes->substitutes
 substituding->substituting
 substitue->substitute
+substitued->substitute, substituted,
```

Comment:
I guess you could also add `substitues->substitutes,` ?

Reply:
And _perhaps_ it makes sense to reorder your suggestions to `substitued->substituted, substitute,` :shrug:

Reply:
c9e99b8